nemo/collections/nlp/models/text_classification/text_classification_model.py

```diff
@@ -85,7 +85,7 @@ def forward(self, input_ids, attention_mask, token_type_ids):
         if isinstance(hidden_states, tuple):
             hidden_states = hidden_states[0]
         logits = self.classifier(hidden_states=hidden_states)
-        return logits
+        return logits.float()
```

Comment:
Should we add for other models too like token_classification?

Reply:
This is workaround for uncertain output format when using autocast - I am still exploring how to avoid it by forcing FP32 on export side. I guess, it may be needed for some Pytorch scenarios as well. Probably, should be flagged as AMP issue.

Reply:
Added reasonable ones.

Reply:
Have you tried setting the precision to 32 within the `.nemo` model config before running export? Megatron models have a precision attribute that is not a part of the trainer and lives inside the model config.